README.md

```diff
@@ -247,6 +247,19 @@ This will create a backup at `/palworld/backups/`
 
 The server will run a save before the backup if rcon is enabled.
 
+## Restore a backup
+
+To restore a backup, use the command:
+
+```bash
+docker exec palworld-server restore
+```
+
+The container must be restarted after the recovery progresses.
+
+> [!IMPORTANT]
+> The `RCON_ENABLED` environment variable must be `true` to use this command.
```

Comment:
Modified readme to be more inline with [Configuring Automatic Updates with Cron](https://github.com/thijsvanloef/palworld-server-docker?tab=readme-ov-file#configuring-automatic-updates-with-cron)
```suggestion
The `RCON_ENABLED` environment variable must be set to `true` to use this command.
> [!IMPORTANT]
> If docker restart is not set to policy `always` or `unless-stopped` then the server will shutdown and will need to be
> manually restarted.
>
> The example docker run command and docker compose file in [How to Use](#how-to-use) already uses the needed policy
```